### DISCUSSION TITLE
OpenTelemetry support

### DISCUSSION THREAD

**Discussion by emanuelef** [eclipse-ditto/ditto#1651] [General]
Hi,
I just wanted to ask since which version OpenTelemetry is supported.

I set the following env vars on ditto things (v2.3.2) but I'm not seeing the traces.

OTEL_EXPORTER_OTLP_ENDPOINT:         satellite.svc.cluster.local:8184                                                                                                                                                       
OTEL_EXPORTER_OTLP_TRACES_PROTOCOL:  grpc                                                                                                                                                                                                   
OTEL_EXPORTER_OTLP_TRACES_INSECURE:  true                                                                                                                                                                                                   
OTEL_METRICS_EXPORTER:               none                                                                                                                                                                                                   
DITTO_TRACING_ENABLED:               true                                                                                                                                                                                                   
DITTO_TRACING_SAMPLER:               always

Do I also need to set service name with OTEL_SERVICE_NAME ?

Thanks

**Comment by thjaeckle** [comment]
Should be available since [2.1.0](https://www.eclipse.org/ditto/release_notes_210.html).
I would however not recommend such an old version, version 3.x resolved a lot of stability and performance issues.

Everything required to configure should be documented at https://www.eclipse.org/ditto/installation-operating.html#tracing
